build.gradle.kts

```
@@ -88,7 +88,7 @@ dependencies {
 
     testImplementation("org.junit.jupiter:junit-jupiter:5.10.0")
     testRuntimeOnly("org.junit.platform:junit-platform-launcher")
-    testImplementation("com.github.tomakehurst:wiremock:2.27.2")
+    testImplementation("com.github.tomakehurst:wiremock:3.0.1")
```

```suggestion
    testImplementation("com.github.tomakehurst:wiremock:3.0.3")
```
https://github.com/wiremock/wiremock/releases/tag/3.0.3

(Reminder just in case Renovate doesn't bump it.)